src/main/java/org/terasology/dynamicCities/sites/SiteFacetProvider.java

```diff
@@ -73,9 +66,10 @@ public void process(GeneratingRegion region) {
                 }
             }
 
-            SiteComponent siteComponent = new SiteComponent(minPos.getX(), minPos.getY());
-
-            siteFacet.setSiteComponent(siteComponent);
+            if (minPos.length() > config.minSpawnGap) {
```

Comment:
Please add a comment here that you assume the spawn to be at (0, 0, 0).

Reply:
Comment has been added in this spot, along with another comment above the configuration setting itself. 👍